content/source/docs/enterprise-beta/api/run.html.md

```diff
@@ -91,12 +91,13 @@ A Run performs a Plan and Apply on the last configuration version created and us
 
 | Method | Path           |
 | :----- | :------------- |
-| POST | /runs |
+| POST | /workspaces/:workspace_id/runs |
 
 ### Parameters
 
 - `id` (`string: <required>`) - specifies the workspace ID to run
 - `is-destroy` (`bool: false`) - specifies if this plan is a destory plan, which will destroy all provisioned resources.
+- `:workspace_id` (`string: <required>`) - specifies the workspace ID where the Run will be executed.
```

Comment:
I'm in favor of using lowercase for terms like "run," where the object in question is a feature (or UI affordance) rather than a product. (Same goes for subsequent instances of this, but I'll only comment on this one so's to not spam.)